pandas/core/arrays/arrow/array.py

```diff
@@ -567,7 +561,7 @@ def dropna(self: ArrowExtensionArrayT) -> ArrowExtensionArrayT:
         -------
         ArrowExtensionArray
         """
-        if pa_version_under6p0:
```

Comment:
Can't this entire `if` statement be removed?

Reply:
Oh must have missed that.